test/shopping-list.test.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+const chai = require('chai');
+const shopping_list_item = require('../js/shopping-list-item.js');
+const shopping_list = require('../js/shopping-list.js');
+const ShoppingListItem = shopping_list_item.ShoppingListItem;
+const ShoppingList = shopping_list.ShoppingList;
+
+const expect = chai.expect;
+const should = chai.should;
+
+describe('ShoppingListItem', function () {
+  it('should be a class', function () {
+    expect(ShoppingListItem).to.be.a('function');
+    expect(new ShoppingListItem()).to.be.instanceof(ShoppingListItem);
+  });
+
+  it("should have property named 'name'", function () {
+    const pizza = new ShoppingListItem('pizza', 'pepperoni');
+
+    expect(new ShoppingListItem()).to.have.property('name');
+    expect(pizza.name).to.be.a('string');
+    expect(pizza.name).to.deep.equal('pizza');
+  });
+
+  it("should have property named 'description'", function () {
+    const pizza = new ShoppingListItem('pizza', 'pepperoni');
+
+    expect(new ShoppingListItem()).to.have.property('description');
+    expect(pizza.description).to.be.a('string');
+    expect(pizza.description).to.deep.equal('pepperoni');
+  });
+
+  it("should have property 'isDone'", function () {
+    const pizza = new ShoppingListItem('pizza', 'pepperoni', 'third');
+
+    expect(pizza).to.have.property('isDone');
+    expect(pizza.isDone).to.be.a('boolean');
+    expect(pizza.isDone).to.deep.equal(false);
+  });
+
+  it('should have a constructor method that accepts 2 arguments, name and description', function () {
+    const pizza = new ShoppingListItem('pizza', 'pepperoni', 'third');
+
+    expect(pizza.name).to.deep.equal('pizza');
+    expect(pizza.description).to.deep.equal('pepperoni');
+    expect(pizza.third).to.be.undefined;
```

Comment:
You should write more tests that determine what happens if you pass in no variables or one variables.